build/package/Installer.DEB.proj

```diff
@@ -107,6 +104,7 @@
       <Exec Command="sudo dpkg -r $(SdkDebianPackageName)" />
       <Exec Command="sudo dpkg -r $(AspNetCoreSharedFxDebianPackageName)" />
       <Exec Command="sudo dpkg -r $(SharedFxDebianPackageName)" />
+      <Exec Command="sudo dpkg -r $(HostFxrDebianPackageNameLongForm)" />
```

Comment:
Could we log an issue to remember to remove this line after https://github.com/dotnet/core-setup/issues/4362 is resolved?

This workaround is ok, but I hope to make sure this is not permanent or we are hiding bugs